src/js/utils/css.js

```diff
@@ -1,16 +1,9 @@
 define([
-    'utils/strings'
-], function(Strings) {
+    'utils/strings',
+    'simple-style-loader/addStyles'
+], function(Strings, styleLoader) {
 
-    var _styleRules = {},
-        _styleSheet;
-
-    var _css = function(selector, styles) {
-        if (!_styleSheet) {
-            _styleSheet = document.createElement('style');
-            _styleSheet.type = 'text/css';
-            document.getElementsByTagName('head')[0].appendChild(_styleSheet);
-        }
+    var _css = function(playerId, selector, styles) {
```

Comment:
Changing the signature of `utils.css` is a no-no, unless you are adding a new param. There are developers that use out utils even though they are undoc'ed. Make `playerId` a new param, and let the style loader append the global sheet if it's undefined.